### PR TITLE
Patch 1 for domcom

### PR DESCRIPTION
**Pull request by taijiweb**
# domcom 
domcom is a web framework to provide dom component

## Features
* the value of any property and other stuffs can be a function
The stuffs, like dom property, the test condition of If, Case component, the items of Repeat component, can be functions. Different from the usage of function as property value in jQuery, the functions are used to declares the computating method for every rendering.
* update only the difference between the dom and the cache
* skip the inative component when updating dom
* decouple with model and controller
* composable component
* the root of the component may be multiple dom elements

### CODE REVIEW

File: files/domcom/update.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "packageManager": "github",
+  "name": "domcom",
+  "repo": "taijiweb/domcom",
+  "basePath": "dist/",
+  "files": {
+    "include": ["domcom.js", "domcom.min.js"]
```
Comment (jimaek) [inline files/domcom/update.json]: Can you remove `domcom.js`? I dont think its required in production

**Comment by taijiweb** [comment]
@jimaek , thanks for your review. I've removed domcom.js, could you please to check it again?

**Comment by jimaek** [comment]
@jsdelivrbot squash and merge

**Comment by taijiweb** [comment]
@jimaek , I'm glad to see that my project was accepted, thank you very much. But I could not access the JS file from the cdn, anything else need to be done? I'm expecting your help. I tried the urls like below: http://cdn.jsdelivr.net/domcom/0.0.1/domcom.min.js
http://cdn.jsdelivr.net/domcom/domcom.min.js

**Comment by jimaek** [comment]
`"basePath": "dist/",` needs to be inside `files` https://github.com/jsdelivr/libgrabber#examples

thats why it failed to update

**Pull request closed without merging**